moped-toolbox/append_srts_desc_to_components_desc/secrets_template.py

```diff
@@ -0,0 +1,12 @@
+HASURA = {
+    "HASURA_GRAPHQL_ENDPOINT": {
```

Comment:
the secrets template var names don't match up with the ones in the script, its missing GRAPHQL

Reply:
👀 Ahhhhh, thanks for catching this! I fixed it in my real secrets file but not the template. 🙏